packages/jest-validate/src/condition.js

```diff
@@ -34,6 +34,6 @@ export function validationCondition(option: any, validOption: any): boolean {
   return getValues(validOption).some(e => validationConditionSingle(option, e));
 }
 
-export function MultipleValidOptions(...args: Array<any>) {
```

Comment:
really?

Reply:
This is re: my previous comment. Argh. No this is wrong: https://flow.org/en/docs/types/functions/#toc-function-declarations

Yet after I rebased & changed the syntax it did not work so I got confused and thought I had it wrong before... maybe flow was just stuck. I'll figure this out.